rust-keybroker/keybroker-server/src/policy.rs

```diff
@@ -14,7 +14,7 @@ pub static MEDIATYPES_TO_POLICY: Map<&'static str, (&'static str, &'static str)>
 pub(crate) fn rego_eval(
     policy: &str,
     policy_rule: &str,
-    reference_values: &str,
+    reference_values: &Option<String>,
```

Comment:
I don't think we want the code to get this far with reference values as `Option<>`. I think we need to gate this at an earlier point in the code, and not even attempt the policy evaluation when the reference values are missing. The absence of a reference value is a very distinct situation, compared to the many other reasons why the policy eval might return a non-`true` result.

Reply:
This has been hopefully addressed with the next patch.